src/test/java/uk/gov/digital/ho/pttg/application/IncomeSummaryContextTest.java

```diff
@@ -407,7 +418,43 @@ public void selfAssessmentSelfEmploymentIncome_nullArg_throwsException() {
 
     @Test
     public void selfAssessmentSelfEmploymentIncome_nonNullArg_setsProperty() {
-        incomeSummaryContext.selfAssessmentSelfEmploymentIncome(Arrays.asList(new AnnualSelfAssessmentTaxReturn(null, null, null)));
-        assertThat(incomeSummaryContext.selfAssessmentSelfEmploymentIncome()).isEqualTo(Arrays.asList(new AnnualSelfAssessmentTaxReturn(null, null, null)));
+        incomeSummaryContext.selfAssessmentSelfEmploymentIncome(asList(new AnnualSelfAssessmentTaxReturn(null, null, null)));
+        assertThat(incomeSummaryContext.selfAssessmentSelfEmploymentIncome()).isEqualTo(asList(new AnnualSelfAssessmentTaxReturn(null, null, null)));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void createEmployerPaymentRefMap_employerlessEmployment_noNpe() {
```

Comment:
Nit: I'd prefer this to say what it does do rather than what it doesn't (i.e. can create without employer)

Reply:
Good point. I'll change.